tests/test_ble_interface_core.py

```
@@ -1036,6 +1042,129 @@ def _factory_without_kwargs() -> _FakeDiscoveryClient:
+
+    awaitable = _AwaitableClose()
+
+    class _Client:
+        def close(self) -> Any:
+            return awaitable
+
+    class _Loop:
+        @staticmethod
+        def create_task(_task: Any) -> None:
+            _task.close()
+            raise RuntimeError("cannot schedule task")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)